src/Facebook/InstantArticles/Transformer/Rules/ConfigurationSelectorRule.php

```diff
@@ -153,16 +159,93 @@ public function findAll($node, $selector)
      * @param $node
      * @return null
      */
-    public function getProperty($property_name, $node)
+    public function getProperty(string $property_name, \DOMNode $node): mixed
```

Comment:
The comment says `@return null`. Let's fix the comment.

Reply:
Fixed! Nice catch!